themes/hugo-theme-ephemeralenvironments/static/js/subscribe.js

```diff
@@ -1,27 +1,40 @@
 var userEmail = document.getElementById("email");
 var submitButton = document.getElementById("submit-btn");
+var submissionStatus = document.getElementById("submission-status");
+var grecaptcha = document.getElementById("recaptcha");
 
 document
   .getElementById("subscribe-form")
   .addEventListener("submit", function (event) {
-    event.preventDefault(); // Prevent the form from submitting normally
+    event.preventDefault();
+
+    let response;
+    try {
+      response = grecaptcha.getResponse(1);
```

Comment:
This `try catch` block was some logic I implemented on the SY site to allow multiple recaptchas per page, however I recently reverted it. Since we only will ever have 1 per page on EE, can we just use `getResponse()` with no params?

Reply:
lol no problem, I actually had that refactored, and figured it was for the multiple recaptcha. But got distracted when the recaptcha was passing without checking it due to the cached browswer history, took a bit to figure that out. Put it back to "normal" state when I cleared the browser history and cache. I'll fix it now.